builtin/common/misc_helpers.lua

```diff
@@ -735,4 +719,4 @@ function core.privs_to_string(privs, delim)
 end
 
 assert(core.string_to_privs("a,b").b == true)
-assert(core.privs_to_string({a=true,b=true}) == "a,b")
+assert(core.privs_to_string({a=true,b=true}) == "a,b")
```

Comment:
Thank you for making me aware of what is causing that.  I noticed that it was the exact same line but I didn't realize it was caused by no newline at the end.